selective_certificate_disclosure/src/utils.rs

```diff
@@ -43,8 +43,8 @@ pub fn safe_indy_check<T>(
 ) -> Result<T, WedprError> {
     match result {
         Ok(v) => Ok(v),
-        Err(e) => {
-            wedpr_println!("Encounter IndyCryptoError: {:?}", e);
+        Err(_e) => {
```

Comment:
Why change this?

Reply:
When running the demo, if the predicate is not satisfied, it will be displayed as follows, so i deleted.
![image](https://user-images.githubusercontent.com/50904634/104865715-afc78180-5977-11eb-9c28-9f264320b73a.png)